tajo-client/src/main/java/org/apache/tajo/client/QueryClient.java

```diff
@@ -94,11 +97,11 @@
    *
    * @return If failed, return null.
    */
-  ResultSet executeQueryAndGetResult(final String sql) throws SQLException;
+  ResultSet executeQueryAndGetResult(final String sql) throws TajoException;
```

Comment:
It is difficult to find the reason why ```executeQueryAndGetResult()``` throws ```TajoException```, but ```executeQuery()``` does not. Would you give some backgrounds?